provider/aiven/client.go

```diff
@@ -166,19 +166,27 @@ func (a *HttpClient) GetServiceConnectionDetails(params *GetServiceInput) (strin
 	return host, port, nil
 }
 
-func (a *HttpClient) DeleteService(params *DeleteServiceInput) (string, error) {
+type DeletionErrNotFound struct {}
```

Comment:
#bikeshed: As I dug into this implementation pattern, a bunch of implementations I found (including the upstream of this broker) use the naming pattern`^ErrFooBar` to make their intention clear to the casual reader who's looking at code not particularly close to the implementation - e.g. https://github.com/alphagov/paas-aiven-broker/pull/9/files#diff-bb23cef78e1b7b1d88ed749b84948c08R234

I have no strong opinion on this :)

Reply:
I like it :) Done.